mix.exs

```diff
@@ -106,6 +106,7 @@ defmodule Backpex.MixProject do
       "guides/live_resource/panels.md",
       "guides/live_resource/fluid-layout.md",
       "guides/live_resource/listen-to-pubsub-events.md",
+      "guides/live_resource/custom_button_labels.md",
```

Comment:
We have a lot of short really specific custom guides. Maybe combine with translations?

Reply:
Moved it to translations and renamed translations to "Custom Labels and Translations"